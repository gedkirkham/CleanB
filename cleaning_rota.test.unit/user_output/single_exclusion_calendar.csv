Date,Kitchen,Bedroom_1,Bedroom_2,Bedroom_3,Bedroom_4,Bedroom_5,Lounge,Garage,Master Bedroom,
01/09/2018,,,,Ali,Razvan,Ali,,,,
08/09/2018,Razvan,Ali,Razvan,Razvan,,,Razvan,Ali,Razvan,
15/09/2018,Ali,,,Ali,Ali,,,,Ali,
22/09/2018,Razvan,Razvan,,Razvan,,,,,Razvan,
29/09/2018,Ali,,,Ali,Razvan,Razvan,,,Ali,
06/10/2018,Razvan,Ali,Ali,Razvan,,,Ali,Razvan,Razvan,
13/10/2018,Ali,,,Ali,Ali,,,,Ali,
20/10/2018,Razvan,Razvan,,Razvan,,,,,Razvan,
27/10/2018,Ali,,,Ali,Razvan,Ali,,,Ali,
03/11/2018,Razvan,Ali,Razvan,Razvan,,,Razvan,Ali,Razvan,
10/11/2018,Ali,,,Ali,Ali,,,,Ali,
17/11/2018,Razvan,Razvan,,Razvan,,,,,Razvan,
24/11/2018,Ali,,,Ali,Razvan,Razvan,,,Ali,
01/12/2018,Razvan,Ali,Ali,Razvan,,,Ali,Razvan,Razvan,
08/12/2018,Ali,,,Ali,Ali,,,,Ali,
15/12/2018,Razvan,Razvan,,Razvan,,,,,Razvan,
22/12/2018,Ali,,,Ali,Razvan,Ali,,,Ali,
