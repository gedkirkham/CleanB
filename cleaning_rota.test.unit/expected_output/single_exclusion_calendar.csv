Date,Kitchen,Bedroom_1,Bedroom_2,Bedroom_3,Bedroom_4,Bedroom_5,Lounge,Garage,Master Bedroom
01/09/2018,,,,Ged,Razvan,Ali,,,
08/09/2018,Razvan,Ali,Ged,Razvan,,,Ged,Razvan,Ali
15/09/2018,Ali,,,Ali,Ali,,,,Ged
22/09/2018,Razvan,Ged,,Ged,,,,,Razvan
29/09/2018,Ali,,,Razvan,Ged,Ged,,,Ali
06/10/2018,Razvan,Razvan,Razvan,Ali,,,Razvan,Ali,Ged
13/10/2018,Ali,,,Ged,Razvan,,,,Razvan
20/10/2018,Razvan,Ali,,Razvan,,,,,Ali
27/10/2018,Ali,,,Ali,Ali,Razvan,,,Ged
03/11/2018,Razvan,Ged,Ali,Ged,,,Ali,Ged,Razvan
10/11/2018,Ali,,,Razvan,Ged,,,,Ali
17/11/2018,Razvan,Razvan,,Ali,,,,,Ged
24/11/2018,Ali,,,Ged,Razvan,Ali,,,Razvan
01/12/2018,Razvan,Ali,Ged,Razvan,,,Ged,Razvan,Ali
08/12/2018,Ali,,,Ali,Ali,,,,Ged
15/12/2018,Razvan,Razvan,,Ged,,,,,Razvan
22/12/2018,Ali,,,Razvan,Ged,Ged,,,Ali
